ansible/testapp-update.yaml

```diff
@@ -1,26 +1,12 @@
 - name: update testapp
-  hosts: validators,loadrunners,ephemeral
```

Comment:
The only relevant task in this playbook for loadrunners is `tasks/testapp-clone-repo.yaml`, which is already included `loader-init.yaml`.